### PR TITLE
feat: add env vars configurable for importer

### PR DESCRIPTION
**Pull request by Hugome**
Changes in this pull request:
- add the ability to configure env variables for the importer (For nordigen, spectre, ...)

### CODE REVIEW

**Comment by JC5** [comment]
Nice work, I'll ask for a review. @morremeyer, could you check this out?

Only comment I have is that your "non-secret" env var examples are definitely secret values.

**Comment by Hugome** [comment]
That's absolutely true, especially with open banking secrets..., nice catch.
I've changed that to some more non sensitive vars.

File: charts/importer/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: importer
-version: 1.1.3
```
Comment (morremeyer) [inline charts/importer/Chart.yaml:3]: Please bump this to 1.2.0 as it adds a new feature :)

File: charts/importer/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{{- if .Values.config.env }}
+apiVersion: v1
+kind: ConfigMap
```
Comment (morremeyer) [inline charts/importer/templates/configmap.yaml:3]: Did you consider using a secret so that it could also contain secrets?

**Review by morremeyer — CHANGES_REQUESTED** [review]
Left a comment for version bumping.

The change in itself is good, I would consider using a Secret instead of a ConfigMap so that it’s acceptable to add secrets to it like you did in the values.yaml as an example in the first iteration.

This would allow to directly template environment variables to secrets without having to create a Secret manually and using `existingSecret`.

File: charts/importer/Chart.yaml (hunk 1 above)
Comment (Hugome) [reply charts/importer/Chart.yaml:3]: I was hesitant on that one, changes done :)

**Comment by Hugome** [comment]
For the `ConfigMap`/`Secret` thing, I used the same `values` logic as other subcharts.
But I understand your idea, so it doesn't require to create a `Secret` first and then deploy the chart.
I honestly don't know which one is the best approach, keeping the same logic as other subcharts or to use a `Secret` instead of a `ConfigMap` ? Any thoughts ?

@morremeyer

**Comment by morremeyer** [comment]
Good that you pointed that out. Let’s keep the same logic as in the other subcharts, this makes them easier to maintain.

**Review by morremeyer — APPROVED** [review]
@JC5 you can go ahead and merge :) 

@Hugome Thanks for the contribution!

**Comment by JC5** [comment]
Thanks all, much appreciated as always!

**Pull request merged**